tests/test_overrides_parser.py

```diff
@@ -52,6 +52,11 @@ def eq(item1: Any, item2: Any) -> bool:
         pytest.param("a b c\t-\t1 2 3", "a b c\t-\t1 2 3", id="value:str-ws-in"),
         pytest.param(" abc-123 ", "abc-123", id="value:str-ws-out"),
         pytest.param("123abc", "123abc", id="value:str-int-id"),
+        pytest.param(r"a\,b", "a,b", id="value:str-esc-comma"),
+        pytest.param(r"\ ab", " ab", id="value:str-esc-space"),
+        pytest.param("ab\\\t", "ab\t", id="value:str-esc-tab"),
+        pytest.param("ab\\\\", "ab\\", id="value:str-esc-backslash"),
+        pytest.param(r"\,", ",", id="value:str-esc-comma-alone"),
```

Comment:
They are easier to read IMO (due to not having to double the \) -- it's very minor though so I can definitely get rid of them

Reply:
just for "\," ?

Reply:
I'm using r strings in the first and last lines you quoted. The reason why I'm not using them in the two other lines is because it's not possible:
* Since I want a tab character with `\t` I can't use r strings (otherwise it would become the string `\t` instead of a tab)
* The second one is less obvious: you can't have an r string that ends with `\"`, because it still believes you're trying to escape the quote.

Reply:
Btw Hydra doc added in f7ccd1e

Reply:
I never used r-strings like that.
ipython seems to thinks these are the exact same strings:
```python
In [3]: r"\ ab" == "\ ab"
Out[3]: True

In [4]: r"\," == "\,"
Out[4]: True
```

Reply:
Oh right, it's true that if the backslash is not followed by a character that makes it a special escape, then Python just accepts it as a backslash. Note however that this triggers a deprecation warning under the hood (you can verify it by calling first `warnings.filterwarnings("default", category=DeprecationWarning)`)

Reply:
gotcha.
fucking stealthy deprecation warnings. such an idiotic design.